salt/netapi/rest_tornado/saltnado.py

```diff
@@ -962,81 +964,86 @@ def _disbatch_local(self, chunk):
                     pass
             raise tornado.gen.Return('No minions matched the target. No command was sent, no jid was assigned.')
 
+        # Map of minion_id -> returned for all minions we think we need to wait on
+        minions = {}
+        for m in pub_data['minions']:
+            minions[m] = False
```

Comment:
we can just do

```
minions = {minion: False for minion in pub_data['minions']}
```

Reply:
oh, did we drop python 2.6 support? IIRC dict comprehension isn't in 2.6

Reply:
yup, we dropped 2.6 starting with 2017.7.0

Reply:
Updated :)